src/data/python-quiz.ts

```diff
@@ -1143,6 +1143,16 @@ const pythonQuiz = [
     Explanation: "+ is used to concatenate and * is used to multiply strings", 
     Link: "https://www.freecodecamp.org/news/learn-python-by-thinking-in-types", 
   },
+  {
+    Question: "Which sorting algorithm is used for built-in sort method of Python?",
+    Answer: "Tim Sort",
+    Distractor1: "Quick Sort",
+    Distractor2: "Merge Sort",
+    Distractor3: "Tim Sort",
```

Comment:
distractors are supposed to be for wrong answers only.
Please choose another option here